pkg/cluster/operation/destroy.go

```diff
@@ -237,10 +237,20 @@ func DestroyMonitored(getter ExecutorGetter, inst spec.Instance, options *spec.M
 func CleanupComponent(getter ExecutorGetter, instances []spec.Instance, cls spec.Topology, options Options) error {
 	retainDataRoles := set.NewStringSet(options.RetainDataRoles...)
 	retainDataNodes := set.NewStringSet(options.RetainDataNodes...)
+	cleanDataNodes := set.NewStringSet(options.CleanDataNodes...)
```

Comment:
What's the purpose? `destroy` is intended to delete *all nodes* of a cluster, it make no sense to "clean data and configs for one node", that's what `scale-in` would do.